src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs

```diff
@@ -371,9 +371,38 @@ private void UpdateInputScope(InputScope inputScope)
 
 		partial void OnTextWrappingChangedPartial(DependencyPropertyChangedEventArgs e)
 		{
-			//TODO : see bug #8178
+			if (e.NewValue is TextWrapping textWrapping && !string.IsNullOrEmpty(Text) &&
+				(textWrapping is TextWrapping.Wrap || textWrapping is TextWrapping.WrapWholeWords))
```

Comment:
*Not much important:*

If the repository uses C# 9 or higher, I *think* this could be simplified with `or` patterns to:

```suggestion
			if (!string.IsNullOrEmpty(Text) &&
				(e.NewValue is TextWrapping.Wrap or TextWrapping.WrapWholeWords))
```